wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -510,19 +510,44 @@ def page_bulk_action_choices(context):
         bulk_actions_list.append(action)
     button_hooks = hooks.get_hooks('construct_page_bulk_action_choices')
     for hook in button_hooks:
-        bulk_actions_list = hook(context.request, bulk_actions_list)
+        hook(context.request, bulk_actions_list)
 
     bulk_actions_list.sort(key=lambda x: x.action_priority)
 
+    bulk_action_more_list = []
+    if len(bulk_actions_list) > 4:
+        bulk_action_more_list = bulk_actions_list[4:]
+        bulk_actions_list = bulk_actions_list[:4]
+
     bulk_action_buttons = [
         PageListingButton(
             action.display_name,
             reverse('wagtailadmin_bulk_action', args=[action.action_type]) + '?' + urlencode({'next': action.next_url}),
             attrs={'aria-label': action.aria_label},
-            priority=action.action_priority
+            priority=action.action_priority,
+            classes=action.classes
         ) for action in bulk_actions_list
     ]
 
+    if bulk_action_more_list:
+        more_button = ButtonWithDropdown(
+            label=_("MORE"),
```

Comment:
```suggestion
            label=_("More"),
```

Reply:
I kept it as 'MORE' to match the style of the page listing actions, where it is achieved using css

Reply:
Could we do the same thing with css and commit Dan’s suggestion? Screen readers tend to spell words written in all capital letters, which is a bit confusing.